src/nvim/ops.c

```diff
@@ -1169,8 +1162,7 @@ static void stuffescaped(char_u *arg, int literally)
  * If "regname" is a special register, return TRUE and store a pointer to its
  * value in "argp".
  */
-int 
-get_spec_reg (
+int get_spec_reg(
```

Comment:
this is indeed my doing and not the linter's idea. I'll refrain from doing this sort of change in the future.